Tests.PubSubTester/Tests.PubSubTester/Controllers/PubSubController.cs

```diff
@@ -53,6 +54,20 @@ public IActionResult Callback(string id, [FromBody] string message)
             return Ok();
         }
 
+        [HttpPost]
+        [Route("dataProcessorChange")]
+        public IActionResult Callback([FromBody] ExpectedDTO<DataProcessorChangeDTO> dto)
```

Comment:
I didn't know I made it to the Tester part, and I was really confused why you introduced these callbacks to what I thought was the "PubSub Controller" 😆